xAPI-Communication.md

```diff
@@ -475,6 +467,9 @@ Stores a single Statement with the given id. POST can also be used to store sing
 
 ###### LRS Requirements
 
+* While an LRS MUST NOT have duplicate Statement Ids due to the uniqueness requirement, it does not have to perform 
+checks at the time Statements are received.  The following requirements are for LRSs that do such a check:
```

Comment:
These requirements apply to all LRSs whether they check or not - they MUST not make modifications to their state. 

The option not to check is already covered by 'Whether it responds with `409 Conflict` or `204 No Content`'.

We can remove these two added lines.